app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -266,6 +266,8 @@ ShortFormApplicationController = (
   $scope.validateHouseholdEligibility = (match, callbackUrl) ->
     $scope.clearHouseholdErrorMessage()
     form = $scope.form.applicationForm
+    if match == 'incomeMatch' && $scope.applicant.household_vouchers_subsidies == 'Yes'
```

Comment:
shouldn't this be `if match == 'incomeMatch' || $scope.applicant.household_vouchers_subsidies == 'Yes'` as opposed to `&&`?